website/pages/api-docs/events.mdx

```diff
@@ -0,0 +1,626 @@
+---
+layout: api
+page_title: Events - HTTP API
+sidebar_title: Events
+description: |-
+  The /event endpoints interact with the local Nomad agent to interact with
+  members and servers.
```

Comment:
How about something like, "The /event endpoints are used to query for and interact with Nomad events"

Reply:
:grimacing: woops, that's a bad copy paste :sweat_smile: